samples/stock/demo/columnrange/demo.html

```diff
@@ -1,6 +1,7 @@
 
 <script src="https://code.highcharts.com/stock/highstock.js"></script>
 <script src="https://code.highcharts.com/stock/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/modules/coloraxis.js"></script>
```

Comment:
Not a bug, but somehow I feel we should use `stock/modules/coloraxis.js` path (the file is the same)